requirements-dev.txt

```diff
@@ -1,5 +1,5 @@
 # Dependencies in Koalas. When you update don't forget to update setup.py and install.rst in docs.
-pandas>=0.23.2,<1.0
+pandas>=0.23.2,<=1.0.1
```

Comment:
Maybe we can just remove the upper bound?

Reply:
@ueshin thanks for the comment!